dataverse-persistence/src/main/resources/db/migration/V16__addedLastChangeForExporterTime.sql

```diff
@@ -3,9 +3,14 @@ BEGIN
   IF EXISTS(SELECT *
     FROM information_schema.columns
     WHERE table_name='dataset' and column_name='guestbookchangetime')
+  AND NOT EXISTS(SELECT *
+    FROM information_schema.columns
+    WHERE table_name='dataset' and column_name='lastchangeforexportertime')
   THEN
       ALTER TABLE dataset RENAME COLUMN guestbookchangetime TO lastchangeforexportertime;
   ELSE
       ALTER TABLE dataset ADD COLUMN IF NOT EXISTS lastchangeforexportertime timestamp;
+      ALTER TABLE dataset DROP COLUMN IF EXISTS guestbookchangetime;
   END IF;
-END $$;
\ No newline at end of file
+END $$;
```

Comment:
This whole sql can be written as:
`ALTER TABLE dataset RENAME COLUMN guestbookchangetime TO lastchangeforexportertime;`
The whole point of keeping db version is to know in what state db was before applying changes to it.
We know from previous versions that `guestbookchangetime` is present